### DISCUSSION TITLE
No Cockpit at all

### DISCUSSION THREAD

**Discussion by Nathaniel57** [flybywiresim/aircraft#3439] [Help] [answered]
![Frontside](https://user-images.githubusercontent.com/78795693/107360019-aee8c200-6ad5-11eb-9b07-95c0570bb790.JPG)
![Backside](https://user-images.githubusercontent.com/78795693/107360020-b019ef00-6ad5-11eb-8db5-e8680fc88c89.JPG)

Hello Guys, i dont know if this has been discussed before. When I load the stable version of the cockpit everything looks good and works for me. Great improvement, thank you for doing that. But if I try the developement or experimental Version, nothing works. The only thing i can see is the window frame of the front windowsand the not yet impemented galley. No Cockpit at all visible. Has this happend to others as well? I tried installing by hand and with the installer, Developement and experimental dont work for me. Thank you for answers.

**Comment by domanek3** [comment, marked as answer]
Hi,
I've got the same effect when I used co-pilot mod.
It worked fine only with stable version and dev version at the very begining (end of January).
So, I had to remove co-pilot mod to see cockpit in current dev and experimental versions.
Slawek

**Reply by Nathaniel57** [reply]
Well thank you very much for that. I kicked the Co and just had my first flight with the flypad from Narita to Osaka. Pretty cool. Thanks and have a great Day.

**Reply by chabon737** [reply]
hello, since yesterday, i have the same problem. no cockpit at all in stable, development or experimental version.
I have deleted all of them and try Flybywire via asobo . the problem remains.
thanks for any help.
Chabon

**Reply by yuriks59** [reply]
download the updated co-pilot mod and everything will work fine, it already exists.

**Reply by chabon737** [reply]
thanks for your answer. where can I find this update please?

**Comment by chabon737** [comment]
Thanks for your answer. But where can i find it?


Le 2 juin 2021 à 19:41, yuriks59 ***@***.***> a écrit :

﻿

download the updated co-pilot mod and everything will work fine, it already exists.

**Reply by yuriks59** [reply]
https://flightsim.to/
